### ISSUE TITLE
Threatening error message

### ISSUE THREAD

**Issue by jamnexus** [skiplagged/pokemongo-php#25]
I am receiving an extremely concerning error message while trying to use the API for my Twitter application.

{"pokemons":[], "error": "This is an illegal request, you are being watched. Either you stop immediately or we will report this to authorities."}

**Comment by aktarer** [comment]
Sorry we had some people really abusing our service. As long as you and your apps aren't making excessive calls, we don't care. We'll lift this block in the next day or so, once we upgrade our systems to handle the high load.

**Comment by morinted** [comment]
Can you share what a recommended rate would be? I've been running every 30 seconds on my own app and wouldn't mind going slower if it helps your load.

**Comment by jamnexus** [comment]
Thanks @aktarer I apologise if the app has been making too many requests. My code could certainly do with tweaking.

**Comment by Raphaelbm** [comment]
I did many requests i'm sorry. I will update my website to make less request.

**Comment by wearelogik** [comment]
Reported to the authorities? Haha... What will you say? "This dude is using my public API excessively?" :P

Thats what we have rate limits for ;)

**Comment by Raphaelbm** [comment]
I have always this error: {"pokemons":[], "error": "This is an illegal request, you are being watched. Either you stop immediately or we will report this to authorities."} I definitely can not call your API ?

**Comment by selfpaidinc** [comment]
I've hit this error after making 5 requests over 10 minutes... I get "cannot create ticket" error when using the PHP class and a threatening error when trying to use the API.

**Comment by selfpaidinc** [comment]
I've managed to get past the ticket issue, now I face "failed to get pdata"

**Comment by wearelogik** [comment]
This project is considered dead.

**Comment by Raphaelbm** [comment]
why ?

**Comment by wearelogik** [comment]
The project creator doesn't say anything, and hasnt since the project started! API has been down for more then 24 hours with absolutely no word from the developer.

Competitiors like PokeVision and PokeHound has shut down after DMCA notices from Niantic, and it's likely that both his android and web-app has gotten the same, to me know, it seems like the Public API has become a Private API instead.

**Comment by morinted** [comment]
> Sorry we had some people really abusing our service. As long as you and your apps aren't making excessive calls, we don't care. We'll lift this block in the next day or so, once we upgrade our systems to handle the high load.

That's a maintainer. The project was down yesterday while they performed maintenance, as the call error informed. I'm running the Python server right now and it's working. [There are still Pokémon on the website](http://skiplagged.com/pokemon)

It's not easy running a suddenly-popular web service, especially when it's not your job/primary function. As far as I'm concerned they are doing fine, and as long as you are not hammering their servers with a single IP over and over you should be able to use their services. It's also rude to insult these people who have provided this service for free without asking for anything back. It's hard to appreciate the time and effort that goes into keeping a live website like that up and running.

Sorry if I got a little ranty, but it's not like we "deserve" their development time. If you don't want to use the project, you can move on to another project.

**Comment by wearelogik** [comment]
Dude, you do realize that what their asking for back is absolutely all the information you scan for ? This project isn't here to help YOU or to be truly open source. This project is here to help SkipLagged crowd-source Pokemons.

I'm solely mentioning that people should stop wasting their time on this library. There is no need to wait and there is no need to be hopeful. I have never seen a more closed open source project then this.

**Comment by morinted** [comment]
If you want to keep your data to yourself, use [PokemonGo-Map](https://github.com/AHAAAAAAA/PokemonGo-Map). I was under the impression that crowd sourcing was the point of this map → by having each trainer use their own account they are harder to take down (as expressed by [this reddit post](https://www.reddit.com/r/pokemongodev/comments/4tm8tm/hey_guys_lets_work_together_on_a_live_map_of_wild/)), and by running a server for my town I can share the data with anyone using the website or application.

**Comment by wearelogik** [comment]
Thats exactly the point. But he's not doing anyone any favors and he's not getting anything back from it like you mentioned. He's getting absolutely all the information :)

But when nothing is working, the developer shuts up and when he does talk, he's being extremely vague and with-holding, things isnt working like they should, and it's impossible for me or you to help in any way because half the code is locked in vault and no information is provided.

**Comment by baskettcase** [comment]
Hi @aktarer if we have been banned for some reason, could you let us know why and I can see if my script is abusing resources etc and modify it... I don't believe it is, but maybe my bounded area is too large? Although it does sound like nobody using the PHP script is able to get in.. so maybe it's something else?

**Comment by Raphaelbm** [comment]
Yes, I have always the threatening error message so it's not my code... Did you blocked my IP or something else ?

**Comment by wearelogik** [comment]
Dont use curl to fetch the result or set the user agent

-----Original Message-----
